speculos/api/events.py

```diff
@@ -7,12 +7,18 @@
 
 from .restful import AppResource
 
+# Approximative minimum vertical distance
```

Comment:
Why all the newlines? The CI linter accepts line until 120 characters, this comment goes until 91 char so it's fine.

Reply:
Updated.